ansible/roles/sufia/tasks/data-repo.yml

```diff
@@ -63,6 +63,31 @@
       dest: '{{ project_app_root }}/app/assets/images/carousel/'
     ignore_errors: True
 
+
+  - name: ensure font directory exists
+    file:
+      path: '{{ project_app_root }}/app/assets/fonts'
+      state: directory
+
+  - name: find old font files
+    find:
+      path: '{{ project_app_root }}/app/assets/fonts/'
+    register: fonts
+
+  - name: delete old font files
+    file:
+      path: '{{ item.path }}'
+      state: absent
+    with_items: '{{ fonts.files }}'
+
+  - name: add new font files
```

Comment:
Similar to the above, if we are adding fonts then do we need to notify a handler to clear the asset cache via the `tmp:clear` rake task or otherwise notify the handler to restart the application?